helm-charts/splunk-otel-collector/values.yaml

```
@@ -28,6 +28,8 @@ splunkPlatform:
   # enables Splunk Platform as a destination.
   endpoint: ""
   # Required for Splunk Enterprise/Cloud (if `endpoint` is specified). Splunk
+  # Alternatively the token can be provided as a secret.
+  # Refer to https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md
```

Would be nice to link to the particular paragraph. Should be this one I believe

```suggestion
  # Refer to https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#provide-tokens-as-a-secret
```

---

Fixed the link in https://github.com/signalfx/splunk-otel-collector-chart/pull/707/commits/4a1ad7dde1e79f08a7dc2357a287ec9f426ff910